gcloud/bigquery/table.py

```diff
@@ -693,6 +699,225 @@ def insert_data(self,
 
         return errors
 
+    @staticmethod
+    def _configure_job_metadata(metadata,  # pylint: disable=R0913
```

Comment:
> Please use disable=too-many-arguments

pylint is fussy about which line the disabling comment is on:  expanding to the more descriptive flag blows 80-columns.

> Why don't we just raise the number of allowed arguments? (Recall the pain we went through to avoid sprinkling these throughout)

I'm fine with raising the limit globally (as you know, I think pylint is over-opinionated on a bunch of these).